system-tests/lib/cre/don/jobs/readcontract/readcontract.go

```diff
@@ -43,11 +43,7 @@ func GenerateJobSpecs(donTopology *cre.DonTopology, chainID int, networkFamily,
 				return nil, errors.Wrap(nodeIDErr, "failed to get node id from labels")
 			}
 
-			if readContractBinaryPath == "" {
```

Comment:
removed bc the invocation will always send the container path at minimum, so this condition will never trigger:
 `filepath.Join(containerPath, readContractBinaryName),`